src/HealthGPS.Tests/LoadNutrientTable.cpp

```diff
@@ -0,0 +1,57 @@
+#include "pch.h"
+#include "program_path.h"
+
+#include "HealthGPS.Core/exception.h"
+#include "HealthGPS/nutrients.h"
+
+#include <filesystem>
+#include <sstream>
+
+using namespace hgps;
+
+namespace {
+constexpr auto *csv_data = R"(food,d,e,a,c,b
+chicken,1,2,3,4,5
+crisps,6,7,8,9,10
+tofu,11,12,13,14,15
+)";
```

Comment:
Why do you need this if you already have the csv data file below for tests?

Reply:
Hmm... Good point. I wrote this test first, because reading from a file in your unit tests involves faffing around with CMake, but then I ended up doing that anyway. So we could just drop this.